specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/subscriptions.json

```diff
@@ -69,6 +65,110 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.StreamAnalytics/locations/{location}/testQuery": {
+      "post": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "operationId": "Subscriptions_TestQuery",
+        "description": "Test the Stream Analytics query before submitting the job.",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "List subscription quota information in West US": {
+            "$ref": "./examples/Subscription_TestQuery.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          }
```

Comment:
How do you pass the query to test?

Reply:
+1 - Seems like there is a missing parameter here